transformer/transformer.go

```diff
@@ -176,11 +184,27 @@ func InitTransformers(transformerToInit map[string]string, selector labels.Selec
 	}
 	transformerConfigs := getFilteredTransformers(transformerToInit, selector, logError)
 	transformerNames := []string{}
-	for transformerName := range transformerConfigs {
+	defaultSelectedTransformerNames := []string{}
+	for transformerName, t := range transformerConfigs {
 		transformerNames = append(transformerNames, transformerName)
+		if v, ok := t.ObjectMeta.Labels[DEFAULT_SELECTED_LABEL]; !ok || cast.ToBool(v) {
+			defaultSelectedTransformerNames = append(defaultSelectedTransformerNames, transformerName)
+		}
 	}
 	sort.Strings(transformerNames)
-	selectedTransformerNames := qaengine.FetchMultiSelectAnswer(common.ConfigTransformerTypesKey, "Select all transformer types that you are interested in:", []string{"Services that don't support any of the transformer types you are interested in will be ignored."}, transformerNames, transformerNames)
+	selectedTransformerNames := qaengine.FetchMultiSelectAnswer(
+		common.ConfigTransformerTypesKey,
+		"Select all transformer types that you are interested in:",
+		[]string{"Services that don't support any of the transformer types you are interested in will be ignored."},
+		defaultSelectedTransformerNames,
+		transformerNames,
+	)
+	deselectedTransformerNames := []string{}
```

Comment:
If we are adding a section for disabled transformers, it might be good to add those disabled because of filters and errors too.

Reply:
done